### ISSUE TITLE
Split workspace problem on file close/open

### ISSUE THREAD

**Issue by jockster74** [pyscripter/pyscripter#814]
Sometimes it is not possible to see a newly opened file. 

I use the split workspace (vertical) feature. Say I have file A to the left and file B to the right.
1.  I close file A.
2. I close file B.
3. I hide the secondary workspace, because I now intend to continue with file C occupying the entire workspace.
4. I double-click on the file C in the File Explorer:
- I can see that a tab  (with the name of file C) is opened where the right-hand side of the split workspace _was_. No content is shown.
- If I drag the tab to the left, it is shown.
- Alternatively, I can split the workspace vertically again (ALT+R), and then the content of file C in the right-hand pane.

In other words, somehow Pyscripter is confused about whether the workspace is split or not when opening file C.
I am using 2.5.3.0 x86 on Windows 7 (64-bit)

**Comment by pyscripter** [comment]
Fixed in version control.
